src/dealDashboard/dealNavbar/dealNavbar.html

```diff
@@ -0,0 +1,3 @@
+<template>
```

Comment:
should the whole template be in a section?

Reply:
updated. I used the component itself as the container. No need for an additional element